transforms/move-pod-description-to-content/move-pod-description-to-content.js

```diff
@@ -0,0 +1,218 @@
+        "argument",
+        "type"
+      ).value;
+
+      if (typeOfSpreadAttributeObject === "Identifier") {
+        // props spread from an object in declaration scope
+        spreadAttributeObject.forEach((attr) => {
+          const descriptionInDeclarationScope = getPropertyFromDeclarationScope(
+            podNode,
+            attr.value,
+            names.descriptionProp
+          );
+
+          if (descriptionInDeclarationScope.size()) {
+            descriptionContent = getDescriptionContent(descriptionInDeclarationScope);
+            descriptionInDeclarationScope.remove();
+          }
+        });
+      } else {
+        const descriptionAsObjectProp = findObjectExpressionsLiteral(
+          j,
+          spreadAttributeObject,
+          names.descriptionProp
+        );
+
+        if (descriptionAsObjectProp.size()) {
+          descriptionContent = getDescriptionContent(descriptionAsObjectProp);
+          descriptionAsObjectProp.remove();
+        }
+
+        const descriptionAsShorthandProp = findObjectExpressionsIdentifier(
+          j,
+          spreadAttributeObject,
+          names.descriptionProp
+        );
+
+        if (descriptionAsShorthandProp.size()) {
+          descriptionContent = getDescriptionContent(descriptionAsShorthandProp);
+          descriptionAsShorthandProp.remove();
+        }
+      }
+    }
+
+    const descriptionOpeningTag = getDescriptionOpeningTag();
+    const descriptionClosingTag = j.jsxClosingElement(
+      j.jsxIdentifier(names.typographyComponent)
+    );
+
+    if (accordionNode.size()) {
+      console.log(accordionNode.paths()[0].value.children)
+      accordionNode.paths()[0].value.children = [
+        descriptionOpeningTag,
+        descriptionContent,
+        descriptionClosingTag,
+        ...accordionNode.paths()[0].value.children,
+      ];
+    } else {    
+      podNode.node.children = [
+        descriptionOpeningTag,
+        descriptionContent,
+        descriptionClosingTag,
+        ...podNode.value.children,
+      ];
+    }
+  });
+
+  function addTypographyImport(j, root) {
+    let typographyImport = root.find(j.ImportDeclaration, {
+      source: {
+        type: "Literal",
+        value: "carbon-react/lib/components/typography",
+      },
+    });
+    const podImport = root.find(j.ImportDeclaration, {
+      source: {
+        type: "Literal",
+        value: "carbon-react/lib/components/pod",
+      },
+    });
+
+    if (typographyImport.size()) {
+      return;
+    }
+
+    typographyImport = j.importDeclaration(
+      [j.importDefaultSpecifier(j.identifier(names.typographyComponent))],
+      j.stringLiteral("carbon-react/lib/components/typography")
+    );
+
+    podImport.insertAfter(typographyImport);
+  }
+
+  function getDescriptionOpeningTag() {
+    return j.jsxOpeningElement(j.jsxIdentifier(names.typographyComponent), [
+      j.jsxAttribute(j.jsxIdentifier("as"), j.stringLiteral("div")),
```

Comment:
Should we also add the `data-element="description"` which is the behaviour from Pod?

Reply:
looks like I have missed this one, people could use that attribute in unit tests so it have to be added